src/masked-input/masked-input.tsx

```diff
@@ -57,7 +58,15 @@ const Component = (
     };
 
     const handleChange = (event: React.ChangeEvent<HTMLInputElement>) => {
-        setUpdatedValue(event.target.value);
+        let value = event.target.value;
```

Comment:
onChange still receives the original `event.target.value`, so you may want to assign the value on the event object